objects.js

```diff
@@ -4827,6 +4827,10 @@ SpriteMorph.prototype.mouseDownLeft = function () {
     return this.receiveUserInteraction('pressed');
 };
 
+SpriteMorph.prototype.mouseScroll = function (y) {
+    return this.receiveUserInteraction('scrolled-' + (y > 0 ? 'up' : 'down'));
```

Comment:
... and - of course - we're missing the value (y) that's been scrolled. But it's actually rather useful even without that. Thanks!